README.md

```diff
@@ -170,7 +170,9 @@ Parameter | Description | Required? | SCM Specific? | Default
 **`repository`** | The repository to push SwaggerHub changes to | yes | - | - 
 **`repositoryOwner`** | The SCM account which owns the above repository | no | `GITHUB`. `BITBUCKET` | - 
 **`scmProject`** | Team Project which contains the target repository | no | `AZURE_DEVOPS_SERVICES` | - 
-**`scmOrganization`** | Specific to Azure DevOps Services, the organization with which to synchronize | no | `AZURE_DEVOPS_SERVICES` | - 
+**`scmOrganization`** | Specific to Azure DevOps Services, the organization with which to synchronize | no | `AZURE_DEVOPS_SERVICES` | -
+**`scmUrl`** | Host URL of the Azure DevOps Server | no | `AZURE_DEVOPS_SERVER` | -
+**`scmProjectCollection`** | Project collection which contains the target repositories project | no | `AZURE_DEVOPS_SERVER` | DefaultCollection
```

Comment:
@michaelmelodysmartbear : CAn shorten to Project collection which contains the target repositories project.

Done